go/master/service.go

```diff
@@ -348,9 +354,9 @@ func (s *Service) GetTask(dummy int, task *Task) error {
 	}
 
 	*task = t.Task
-	log.WithFields(s.logFields()).Infof("Task #%d dispatched.", task.ID)
+	log.WithFields(s.logFields()).Infof("Task #%v dispatched.", t)
```

Comment:
A task contains `[]Chunks`, maybe that is too much information to print.
Perhaps change to `log.WithFields(s.logFields()).Infof("Task #%v dispatched.", t.Task.Meta)`